src/components/integrated/OakLessonInfoCard/OakLessonInfoCard.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+
+import {
+  OakFlex,
+  OakHeading,
+  OakHeadingTag,
+  OakIcon,
+  OakIconName,
+} from "@/components/base";
+import { OakHandDrawnCard, OakPrimaryInvertedButton } from "@/components/ui";
+
+export type LessonInfoCardProps = {
+  iconName: OakIconName;
+  infoCardTitle: string;
+  infoCardDescription: string | string[];
```

Comment:
I'd suggest always opting for `ReactNode` when it comes to content — it's far more flexible and there will come a time when you need to apply some styling to the content.